### DISCUSSION TITLE
AI SDK Playground pricing

### DISCUSSION THREAD

**Discussion by Vicegale** [vercel/community#4238] [Feature Question]
### Question

There are several references to the Pro plan in the AI site:
- The Vercel AI page (https://vercel.com/ai) refers to GPT-4 usage in the playground when using the Pro plan.
- The token limit pop-up refers to the Pro plan (can't recall exact wording, haven't reached the limit today)

However, I was unable to find info on what getting the Pro plan gets you, when it comes to the playground (new token limits, etc). Either in the [docs](https://sdk.vercel.ai/docs), or the [Vercel pricing page](https://vercel.com/pricing). Or that having a Pro Trial is different than a paid plan.

Is this info available anywhere?

**Comment by amyegan** [comment]
Great question, @Vicegale. You can find more info about Pro trials in our [Plans](https://vercel.com/docs/accounts/plans) documentation at https://vercel.com/docs/accounts/plans/pro/trials

The trial generally follows the same limits as a regular plan but with a few specified usage restrictions. You can find the current specific limits at https://vercel.com/docs/accounts/plans/pro/trials#trial-limitations

**Reply by Vicegale** [reply]
Thanks for the answer! However, my question concerns specifically the AI SDK playground. No information on it seems to be listed on the [pricing page](https://vercel.com/pricing) for the Vercel Pro tier.

I'm specifically looking for more info on what the right-most button is mentioning, as well as any limits on the AI Playground:
![image](https://github.com/vercel/community/assets/26018528/6be0c7c4-ba0e-4ac8-ae4b-6c705b89ea32)

**Reply by amyegan** [reply]
I see what you mean. I checked with the AI team to be sure, and they confirmed that a full Pro plan is needed for GPT-4 access. This is one of the trial limitations. 

Since that detail seems to be missing from the docs, I'm opening a ticket to get the information added. Thanks for pointing this out!

**Reply by Vicegale** [reply]
Thanks for checking! Is there any word on token limits when using the full Pro plan?
